src/capi/azext_capi/custom.py

```diff
@@ -309,9 +369,34 @@ def create_workload_cluster(  # pylint: disable=unused-argument,too-many-argumen
         ephemeral_disks=False,
         windows=False,
         pivot=False,
-        output_path=None,
+        template=None,
         yes=False):
 
+    if template:
```

Comment:
This is minor but I wonder if this would read better internally named as `custom_template` or `user_provided_template`.  I like `--template` for the external facing the UX but here and a few other places I think the code would be more descriptive with something like `if user_provided_template`

Reply:
definitely, I like `user_provided_template` for more detailed description in the code